code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -93,6 +94,15 @@
 					var/obj/item/weapon/gun/projectile/automatic/m90/gat
 					gat.pin = /obj/item/device/firing_pin
 					new gat(get_turf(H))
+				if("boltaction")
+					new /obj/item/weapon/gun/projectile/shotgun/boltaction(get_turf(H))
+				if("speargun")
+					new /obj/item/weapon/gun/projectile/automatic/speargun(get_turf(H))
+				if("minigun")
+					var/obj/item/weapon/gun/projectile/automatic/gatling/gat     // This looks like fun addition but it's secretly fun removal

```

Comment:
This have to be fixed (the runtime was present in the original file, see  #8197)